.scrutinizer.yml

```diff
@@ -17,6 +17,7 @@ build:
             - gulp lint
             - ./bin/install-wp-tests.sh wordpress_test root ''
         override:
+            - eslint-run "assets/src/**"
```

Comment:
I was actually going to ask. Not sure how to make it fail on errors and not warnings. Do you know? Or is that automatic?